### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by aflavio** [sunaku/tamzen-font#14]
Hi. 

Do you have some pretension to add more glyphs? Merge tamzen with siji font will be great.

**Comment by sunaku** [comment]
Great idea! :+1: I didn't know about [siji](https://github.com/stark/siji). :joy_cat: 

I collaborated on an [early version of stlarch-font](https://bbs.archlinux.org/viewtopic.php?id=129275) (also [Tamsyn](www.fial.com/~scott/tamsyn-font/) based) so this brings me back full circle. :cyclone:

**Comment by aflavio** [comment]
I can help you with this integration. Did You use fontforge to design this font, right?

**Comment by sunaku** [comment]
No, this font was programatically generated from BDF sources by the `Rakefile` script in this repo. :neckbeard:
